### PR TITLE
[CHI-183] Update Header component with nav tags and include secondary…

### PR DESCRIPTION
**Pull request by davidojedacl**
… top nav 

@mattnickles @jllr    Pending to support dropdown in small tabs

### CODE REVIEW

**Comment by davidojedacl** [comment]
@mattnickles The small tabs have a height of 3rem (48px) .So is necessary to change the small tabs height if the -navbar -top must have 2rem (32px)

**Comment by mattnickles** [comment]
@dani-cl-madrid @davidojedacl any thoughts on creating a subcomponent within o-header called o-header__toolbar or something along those lines and defining this unique tab size there? i don't think it will be common enough to warrant adding a -xs modifier to the tab component, but it will be very common within headers.

This is what we're looking to achieve (tabs on top left):
image

**Comment by mattnickles** [comment]
@davidojedacl can you resolve conflicts?

**Comment by davidojedacl** [comment]
Conflicts solved

File: src/chi/components/header/header.scss [hunk 1]
```diff
@@ -92,27 +89,71 @@
...
 
+    &__toolbar {
+      align-items: center;
+      display: flex;
+      flex-direction: row;
+      height: 2rem;
+      justify-content: center;
+      position: relative;
+      width: 100%;
+      @include respond-to(sm) {
```
Comment (jllr) [inline src/chi/components/header/header.scss]: @davidojedacl for what is this needed?

File: src/chi/components/header/header.scss [hunk 2]
```diff
@@ -92,27 +89,71 @@
...
+        flex-direction: row;
+        justify-content: space-between;
+        padding: 0 1rem;
+      }
+      @include respond-to(md) {
```
Comment (jllr) [inline src/chi/components/header/header.scss]: @davidojedacl for what is this needed?

File: src/chi/components/tabs/tabs.scss [hunk 3]
```diff
@@ -302,6 +302,24 @@
       }
     }
   }
+
+  &.-toolbar {
```
Comment (jllr) [inline src/chi/components/tabs/tabs.scss]: @davidojedacl shouldn't this be done in header scss as this is an specific case of header toolbar? @mattnickles thoughts

**Comment by jllr** [comment]
@davidojedacl there's some extra padding in the markup because of the left tabs. Also the arrow icon is rendering green

screen shot 2019-02-28 at 4 57 39 pm

screen shot 2019-02-28 at 4 59 53 pm

**Comment by davidojedacl** [comment]
done

File: src/chi/components/header/header.scss (hunk 2 above)
Comment (jllr) [reply src/chi/components/header/header.scss]: @davidojedacl it is still in the header.scss

File: src/chi/components/header/header.scss (hunk 1 above)
Comment (jllr) [reply src/chi/components/header/header.scss]: @davidojedacl it is still in the header.scss

File: src/chi/components/header/header.scss [hunk 4]
```diff
@@ -95,16 +95,9 @@
       flex-direction: row;
       height: 2rem;
       justify-content: center;
+      padding: 0 1rem;
       position: relative;
       width: 100%;
-      @include respond-to(sm) {
```
Comment (jllr) [inline src/chi/components/header/header.scss]: @davidojedacl please return this piece of code, as we saw it was in the original header version

Comment (davidojedacl) [reply src/chi/components/header/header.scss]: done

File: src/website/views/components/header.pug [hunk 5]
```diff
@@ -774,7 +774,173 @@ ul#a-tabs--tabs-buttons-avatars.a-tabs
...
+          .o-header__start
+            span.o-header__title.-text--muted Application Title
+          .o-header__end
+            a.a-btn.-flat(href="#") Login
+            a.a-btn.-brand.-ml--1(href="#") Sign Up
```
Comment (mattnickles) [inline src/website/views/components/header.pug]: We should be rendering a mint button on the inverse example. Can you change the button class to `.-secondary` ?

**Pull request merged**